gradle/dependencies.gradle

```diff
@@ -13,15 +13,15 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '129.0.0'
+      mapboxNavigatorVersion = '130.0.0'
```

Comment:
See NN changelog. 
We should extract sth like:
"Supported Hungarian, Finnish, Serbian, Greek, Czech, Slovak and Lithuanian languages for voice instructions in offline routes".